src/base/log.cc

```diff
@@ -82,8 +82,7 @@ class WindowsLogger {
 		std::cout.rdbuf(stdout_.rdbuf());
 		std::cerr.rdbuf(stderr_.rdbuf());
 		// Repeat version info so that we'll have it available in the log file too
-		std::cout << "This is Widelands Version " << build_id() << " (" << build_type() << ")"
-		          << std::endl;
+		std::cout << "This is Widelands Version " << build_ver_details() << std::endl;
```

Comment:
One more in the "consistency" category:
```suggestion
		std::cout << "This is Widelands version " << build_ver_details() << std::endl;
```
The rest looks good now. I don't find any other occurrences of the version.